package.json

```diff
@@ -2,7 +2,7 @@
   "name": "bw-chk",
   "version": "1.1.0",
   "description": "Parses uncompressed Broodwar map files (scenario.chk).",
-  "main": "lib/index.js",
+  "main": "index.js",
```

Comment:
This should actually not be changed ^^

The main entry file *should* be `lib/index.js`, which is the file that's built after `build` script is run. You can see the build script here: https://github.com/ShieldBattery/bw-chk/blob/master/package.json#L10
As you can see, that script uses babel to transpile the modern javascript code into an older version of javascript code.

The problem that you probably encountered is that you haven't run the `build` script, which is a bit cumbersome to do manually, so we should probably add a way to run that script automatically each time before publishing to npm. That can be easily done by declaring a script called `prepublishOnly`. An example of it can be found here:
https://github.com/ShieldBattery/node-interval-tree/blob/master/package.json#L14